application/dash_application/views/single_lane_wgs.py

```diff
@@ -160,8 +158,6 @@ def get_wgs_data():
                 bamqc_df[FASTQC_COL.TotalSequences] *
                 bamqc_df[BAMQC_COL.AverageReadLength] / 1e9)
         , 3)
-    ichorcna_df[special_cols["Purity"]] = round(
```

Comment:
You can also remove loading of data (133-138) and merging 163-167)

Reply:
Done!